sandbox/farm/animals/monkey.js

```diff
@@ -0,0 +1,52 @@
+import { Animal } from './animal.js';
+
+const movementCycleMs = 4000;
+
+export function Monkey(canvas) {
+  this.phase = Math.random() * Math.PI * 2;
+  this.pixels = [
+    { x: 0, y: 0 },
+    { x: -1, y: 0 },
+    { x: 1, y: 0 },
+    { x: 0, y: 1 },
+    { x: -1, y: 1 },
+    { x: 1, y: 1 },
+    { x: -2, y: 1 },
+    { x: 2, y: 1 },
+    { x: 0, y: 2 },
+    { x: 1, y: 2 }, 
+    { x: -1, y: 2 },
+    { x: 0, y: 3 },
+    { x: 0, y: 4 },
+    { x: -1, y: 4 },
+    { x: 1, y: 4 },
+    { x: -1, y: 4 },
+    { x: 1, y: 4 },
+    { x: -2, y: 4 },
+    { x: 2, y: 4 },
+    { x: -3, y: 4 },
+    { x: 3, y: 4 },
+    { x: 0, y: 5 },
+    { x: -1, y: 5 },
+    { x: 1, y: 5 },
+    { x: 0, y: 6 },
+    { x: -1, y: 6 },
+    { x: 1, y: 6 },
+    { x: 2, y: 6 },
+    { x: 1, y: 7 },
+    { x: -1, y: 7 },
+    { x: 3, y: 7 },
+    { x: 1, y: 8 },
+    { x: -1, y: 8 },
+    { x: 4, y: 8 },
+  ];
+  this.init(canvas);
+}
+
+Monkey.prototype = new Animal();
+
+Monkey.prototype.move = function() {
```

Comment:
Отличная получилась обезьянка) Можно добавить подобную функцию движения, для создания илюзии прыжка)

```suggestion
Monkey.prototype.move = function() {
  const angle = ((Date.now() % movementCycleMs) * 2 * Math.PI / movementCycleMs) + this.phase;
  this._x = this._x + Math.cos(angle) / 10;
  this._y = this._y + Math.cos(angle) / 2;
}
```